start/src/main/java/org/apache/accumulo/start/classloader/vfs/AccumuloReloadingVFSClassLoader.java

```diff
@@ -199,6 +200,9 @@ public AccumuloReloadingVFSClassLoader(String uris, FileSystemManager vfs,
    * thread
    */
   public void close() {
+    for (FileObject file : Arrays.asList(this.files)) {
+      monitor.removeFile(file);
+    }
```

Comment:
Could do something like: `Stream.of(this.files).forEach(monitor::removeFile);` instead.